docs/adding-ads.md

```diff
@@ -0,0 +1,23 @@
+# Adding Super Cool Ads for Display in Posts
+
+After you [configure the plugin](configuration.md) to set the number of paragraphs before each ad insertion, and the number of available ad insertions, visit **Appearance** > **Widgets** to create the ads themselves. You'll see a number of Inserted Ad Positions as Widget areas where you can place widgets. You can set the number of available Inserted Ad Positions in the [plugin options panel](configuration.md). Each Inserted Ad Position widget area can have one and only one ad.
```

Comment:
Could change the last line to say "Each Inserted Ad Position should have only one ad."